Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends gconf-service l
                                                 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates \
                                                 fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils \
                                                 x11vnc x11-xkb-utils xfonts-100dpi xfonts-75dpi xfonts-scalable \
-                                                xfonts-cyrillic x11-apps xvfb xauth wget netcat \
+                                                xfonts-cyrillic x11-apps xvfb xauth wget netcat dumb-init \
```

Comment:
[`dumb-init`](https://engineeringblog.yelp.com/2016/01/dumb-init-an-init-for-docker.html) looks interesting, but I don't follow what all it's doing or why it's used here. Something about the process being killed because of yarn cache reasons?

Reply:
There's an older but good writeup on what this does and why it's good to have: https://engineeringblog.yelp.com/2016/01/dumb-init-an-init-for-docker.html

Reply:
As someone who also was not familiar with `dumb-init`, I gathered my own tl;dr - 

- docker containers often only have 1 running process
- PID 1 is special
- you dont want your app running as PID 1
- `dumb-init` runs as PID 1 and gracefully handles kernal/system level commands?